src/components/TextInputFocusable/index.js

```diff
@@ -179,7 +183,11 @@ class TextInputFocusable extends React.Component {
     }
 
     focusInput() {
-        this.textInput.focus();
+        if (this.props.isFocusable) {
+            this.textInput.focus();
+        } else {
+            this.textInput.blur();
```

Comment:
Not sure why it would be necessary to blur this. Is preventing focus not enough?

Reply:
@marcaaron It's not enough, when testing live on safari if I pass the `isFocusable = false` as a prop, `this.textInput.focus();` gets called, this may be a bug in react-native-web, I had to add `this.textInput.blur();` which removes the focus to make sure it applied.
`isFocusable `is an optional input and is true by default.